src/organisation/containers/organisation/organisation.component.html

```diff
@@ -4,26 +4,28 @@ <h1 class="govuk-heading-xl"> Organisation</h1>
      <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
          
-     <!-- <table class="govuk-table">
+     <table class="govuk-table">
         <tbody class="govuk-table__body">
            <tr class="govuk-table__row">
               <th class="govuk-table__header" scope="row">Name</th>
-              <td class="govuk-table__cell">{{orgData.name}}</td>
+              <td class="govuk-table__cell">{{orgData?.name}}</td>
               <td class="govuk-table__cell"></td>
            </tr>
            <tr class="govuk-table__row">
               <th class="govuk-table__header" scope="row">Address</th>
-              <td class="govuk-table__cell">{{orgData.address1}}<br>{{orgData.townCity}}<br>{{orgData.postcode}}</td>
+              <td class="govuk-table__cell">
+              <span *ngIf="orgData?.houseNoBuildingName">{{orgData?.houseNoBuildingName}}<br></span>
```

Comment:
do u think is possible to do *ngFor here rather then ngIf? to reduce the boiler plate code? maybe  reduce js method could be used to reduce object into array then loop the array to produce <span>... this way if any other properties gets added into future it will just work without any maintenance... ping me directly if u need to discuss it